libraries/shared/src/LogHandler.cpp

```diff
@@ -51,6 +71,29 @@ const char* stringForLogType(LogMsgType msgType) {
     }
 }
 
+const char* colorForLogType(LogMsgType msgType) {
+    switch (msgType) {
+        case LogInfo:
+            return "\u001b[37;1m";  // Bold white
+        case LogDebug:
+            return "";
+        case LogWarning:
+            return "\u001b[35;1m";  // Bright magenta
+        case LogCritical:
+            return "\u001b[31;1m";  // Bright red
+        case LogFatal:
+            return "\u001b[31;1m";  // Bright red
+        case LogSuppressed:
+            return "";
+        default:
+            return "";
+    }
+}
+
+const char* colorReset() {
+    return "\u001b[0m";
```

Comment:
Add a comment saying what colour this is?

Reply:
It's not a color, it resets the color to the default.

Reply:
https://www.lihaoyi.com/post/BuildyourownCommandLinewithANSIescapecodes.html#16-colors